app/Community/Commands/SyncUserUlids.php

```diff
@@ -18,28 +17,22 @@ class SyncUserUlids extends Command
 
     public function handle(): void
     {
-        $total = User::whereNull('ulid')->count();
-
-        if ($total === 0) {
-            $this->info('No records need ULIDs.');
-
-            return;
-        }
+        $total = User::count();
 
         $progressBar = $this->output->createProgressBar($total);
         $progressBar->start();
 
         User::withTrashed()
-            ->whereNull('ulid')
-            ->chunkById(4000, function ($users) use ($progressBar, &$lastTimestamp) {
+            ->chunkById(4000, function ($users) use ($progressBar) {
                 $updates = [];
 
                 /** @var User $user */
                 foreach ($users as $user) {
-                    // Increment timestamp for each ULID to ensure some variance.
-                    $lastTimestamp++;
+                    $milliseconds = rand(1, 20);
```

Comment:
This should be `+=`. We still want to simulate increasing timestamps for entries created in the same second.

Without the `+=`, the `%=` on the next line is unnecessary.

Reply:
Fixed to `+=` in latest.